### ISSUE TITLE
run-pty --auto-exit: Commands can `exit 0` unexpectedly in GitHub Actions

### ISSUE THREAD

**Issue by lydell** [lydell/run-pty#45]
In another project, I use `run-pty --auto-exit` to run 5 checks.

Locally, everything (so far) works perfectly.

But in GitHub Actions, some commands can sometimes exit early with code 0, causing lint errors and suchlike not to be caught in CI.

I need to dig deeper into this. First thing is to try to reproduce it in this repo I think, that’ll make debugging much easier.

**Comment by lydell** [comment]
This seems to be a bug in node-pty: https://github.com/microsoft/node-pty/issues/630

**Comment by lydell** [comment]
Workaround released in v5.0.0.

**Comment by lydell** [comment]
Closing, see here for why: https://github.com/microsoft/node-pty/issues/630#issuecomment-2336404137

TL;DR run-pty does not `exit 0` unexpectedly anymore. The issue isn’t 100 % solved, though, but there is nothing to track in this repo about it for a while.
